src/components/common/popup/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useAppSelector } from "hooks";
+import { popupSelector, PopupType } from "store/reducers/popupReducer";
+import FormPopup from "./FormPopup";
+
+export type PopupProps = {
+  title: string;
+  isOpen: boolean;
+  children: React.ReactNode;
+  closeHandler: () => void;
+};
+
+type PopupMapType = {
+  [PopupType.FORM_POPUP]: React.FC<PopupProps>;
+};
+
+const PopupMap: PopupMapType = {
+  [PopupType.FORM_POPUP]: FormPopup,
+};
+
+const PopupWrapper = () => {
+  const { popupKey, popupProps } = useAppSelector(popupSelector);
+  const Popup = PopupMap[popupKey as keyof PopupMapType];
+
+  return <div>{popupKey ? <Popup {...popupProps} /> : null}</div>;
```

Comment:
```suggestion
  return ({popupKey ? <Popup {...popupProps} /> : null});
```

Reply:
It should be `return popupKey ? <Popup {...popupProps} /> : null;`

Reply:
OK